src/App.tsx

```diff
@@ -1,24 +1,103 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useState, useEffect, useMemo } from 'react';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import * as postService from './api/todos';
+import { Status } from './types/Status';
+import { Errors } from './components/Errors/Errors';
+import { Header } from './components/Header/Header';
+import { TodoList } from './components/TodoList/TodoList';
+import { Footer } from './components/Footer/Footer';
 
-const USER_ID = 0;
+const USER_ID = 11098;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [todosFilterBy, setTodosFilterBy] = useState<Status>(
+    Status.ALL,
+  );
+  const [isError, setIsError] = useState('');
+
+  const showError = (text: string) => {
+    setIsError(text);
+    setTimeout(() => {
```

Comment:
You should clear all timers on the component unmount to avoid unpredictable side effects https://bobbyhadz.com/blog/react-cleartimeout